src/pybind/mgr/snap_schedule/tox.ini

```diff
@@ -0,0 +1,19 @@
+[tox]
+envlist = py27,py3
+skipsdist = true
+; toxworkdir = {env:CEPH_BUILD_DIR}/snap-schedule
+; minversion = 2.8.1
+
+[testenv]
+setenv=
+    LD_LIBRARY_PATH = {toxinidir}/../../../../build/lib
+    PATH = {toxinidir}/../../../../build/bin:$PATH
+    py27: PYTHONPATH = {toxinidir}/../../../../build/lib/cython_modules/lib.2:..
+    py3:  PYTHONPATH = {toxinidir}/../../../../build/lib/cython_modules/lib.3:{toxinidir}
+    CEPH_SNAP_SCHEDULE_UNITTEST = true
+deps =
+    pytest
+    mock
+    py27: pathlib
+commands=
+    pytest {posargs}
```

Comment:
No need to add this. just add yourself to 

https://github.com/ceph/ceph/blob/90699f6e48c322ec8612939617e26df6305b8da1/src/pybind/mgr/tox.ini#L14-L22